### PR TITLE
Add Japanese translation of README

### PR DESCRIPTION
**Pull request by tomato3713**
Added japanese translation of README.

### CODE REVIEW

File: .github/README_jp.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
+<h1 align="center">
+    <a href="https://github.com/Shpota/goxygen/tree/master/.github/README.md">
+        <img height="25px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/gb.svg">
```
Comment (shpotainna) [inline .github/README_jp.md:3]:
```suggestion
        <img height="20px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/gb.svg">
```

File: .github/README_jp.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+        <img height="20px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/by.svg">
+    </a>
+    <a href="https://github.com/Shpota/goxygen/tree/master/.github/README_fr.md">
+        <img height="20px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/fr.svg">
+    </a>
+    <a href="https://github.com/Shpota/goxygen/tree/master/.github/README_es.md">
+        <img height="20px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/es.svg">
+    </a>
+    <a href="https://github.com/Shpota/goxygen/tree/master/.github/README_jp.md">
+        <img height="20px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/jp.svg">
```
Comment (shpotainna) [inline .github/README_jp.md:30]:
```suggestion
        <img height="25px" src="https://cdnjs.cloudflare.com/ajax/libs/flag-icon-css/3.4.6/flags/4x3/jp.svg">
```

**Comment by Shpota** [comment]
@tomato3713 thank you for your contribution! I merged the changes.

I will be adding [relational DB support](https://github.com/Shpota/goxygen/pull/55) soon which will require updating a couple of sentences in the documentation. May I ask you to help adjust the documentation once the corresponding functionality is ready?

**Comment by tomato3713** [comment]
@Shpota  
Thanks for merging.
Yes! Adjust documents when updating.

**Pull request merged**